Scripts/script-EmailAskUser.yml

```diff
@@ -3,68 +3,82 @@ commonfields:
-      demisto.results(res)
-      sys.exit(0)
-  entitlement = demisto.get(res[0], 'Contents')
+  // Get entitlement
+  res = executeCommand('addOneTimeEntitlement', {});
+  if (res[0].Type == entryTypes.error) {
+      return res;
+  }
+  var entitlement = res[0].Contents;
 
-  #### Handle options
-  option1 = demisto.get(demisto.args(), 'option1')
-  if not option1:
-      option1 = 'yes'
-  option2 = demisto.get(demisto.args(), 'option2')
-  if not option2:
-      option2 = 'no'
 
-  #### Create email subject
-  subjectSuffix = ' - #' + demisto.investigation()['id'] + ' ' + entitlement
-  if demisto.get(demisto.args(), 'task'):
-      subjectSuffix += ' #' + demisto.get(demisto.args(), 'task')
+  // Create email subject
+  var subjectSuffix = ' - #' + incidents[0].id + ' ' + entitlement;
+  if (args.task) {
+      subjectSuffix += ' #' + args.task;
+  }
+  var subject = args.subject + subjectSuffix;
 
-  #### Create email body
-  if (bodyType == 'html'):
-      message = '%s<br/><br/>Please reply with either %s or %s' % (demisto.args()['message'], option1, option2)
-  else:
-      message = '%s\n\nPlease reply with either %s or %s' % (demisto.args()['message'].replace('\\n', '\n'), option1, option2)
+  // Handle options
+  var reply = args.replyAddress;
+  var option1 = args.option1;
+  if (!option1) {
+      option1 = 'yes';
+  }
+  var option2 = args.option2;
+  if (!option2) {
+      option2 = 'no';
+  }
+  if (reply && bodyType == 'html') {
+      option1 = '<a href="mailto:' + reply + '?subject=' + subject + '&body=' + option1 + '">' + option1 + '</a>';
+      option2 = '<a href="mailto:' + reply + '?subject=' + subject + '&body=' + option2 + '">' + option2 + '</a>';
+  }
 
-  #### Get email recipients
-  roles = demisto.get(demisto.args(), 'roles')
-  addresses = []
-  if roles is not None:
-            usersRes = demisto.executeCommand('getUsers', { "roles": roles });
-            emails =  map(lambda x: x["email"], usersRes[0]["Contents"])
-            addresses.extend(emails)
-  argMail = demisto.get(demisto.args(), 'email')
-  if argMail is not None:
-      addresses.extend([argMail])
+  // Create email body
+  var message = args.message;
+  if (bodyType == 'html') {
+      message += '<br/><br/>';
+  } else {
+      message += '\n\n';
+  }
+  message += 'Please reply with either ' +option1+' or '+option2;
 
-  if len(addresses) > 0:
-      #### prepare args and run send-mail
-      adStr = ",".join(addresses)
-      args = demisto.args()
-      args['to'] = adStr
-      args['subject'] = demisto.get(demisto.args(), 'subject') + subjectSuffix
-      args['body'] = message
-      if (bodyType == 'html'):
-          args['htmlBody'] = message
-      else:
-          args['body'] = message
+  // Get email recipients
+  var addresses = [];
+  if (args.roles) {
+      var usersRes = executeCommand('getUsers', { "roles": args.roles });
+      if (usersRes[0].Contents.length>0) {
+          addresses.push(dq(usersRes,"Contents.email"));
```

Comment:
I think that the result here can be a simple object and then it will work for you
But it can also be an array, and then you will have array of arrays